scripts/vivado/add_sources.tcl

```diff
@@ -0,0 +1,36 @@
+set ROOT [file normalize [file dirname [info script]]/../..]
+# This script was generated automatically by bender.
```

Comment:
For verification scripts in other repos, we invoke Bender within a script but do not check the output script into Git.  Why did you check this one in?

Reply:
I had to manipulate the bender output, as with the vivado script the `TARGET_SIMULATION` flag is not set.